languages/c/templates/imports/common.c

```diff
@@ -0,0 +1 @@
+#include "JsonData_${info.title}.h"
```

Comment:
common.c & default.c look the same now... is that intentional?

Reply:
common.c is removed